web/app/Http/Controllers/PageController.php

```diff
@@ -36,21 +38,15 @@ public function show(): View
             $sidebar_content = $values['sideBar1Content'] ?? null;
             $page_content = $values['pageContent'] ?? null;
             $tags = $values['tags'] ?? null;
-            $revision = $page->getRevisionNumber();
-            $timestamp = $page->getDateLastEdited()->getTimestamp();
-            $page_title = $page->getTitleOrUnixName();
-            $title = $page_title;
-            $social_title = $page_title;
+            $revision = $page->revision_number;
+            $timestamp = $page->lastUpdated()->getTimestamp();
+            $title = $page->title;
+            $license = LicenseMapping::get('cc_by_sa_3'); // TODO hardcoded
```

Comment:
Is this still meant to be hardcoded?

Reply:
Yes it's a category setting, which doesn't exist anymore, so we're just setting it to until we move license choice into site settings.